pychunkedgraph/app/segmentation/v1/routes.py

```diff
@@ -204,6 +204,19 @@ def change_log_full(table_id):
     return output
 
 
+@bp.route("/table/<table_id>/tabular_change_log_recent", methods=["GET"])
+@auth_requires_permission("view")
+def tabular_change_log_weekly(table_id):
+    start_time = request.args.get("start_time", default=0, type=int)
```

Comment:
we try to move as much logic as possible to `common.py`. In general, we use this format to extract timestamps:

```
    try:
        timestamp = float(request.args.get("timestamp", time.time()))
        timestamp = datetime.fromtimestamp(timestamp, UTC)
    except (TypeError, ValueError) as e:
        raise (
            cg_exceptions.BadRequest(
                "Timestamp parameter is not a valid" " unix timestamp"
            )
        )
```